dev

```diff
@@ -19,51 +20,22 @@ LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.EnvMarketplaceToken=0x5861
 
 MESG_SERVER_PORT=${MESG_SERVER_PORT:-50052}
 
-# if set to true the docker with system services will be started.
-MESG_SERVICE_SERVER_ON=${MESG_SERVICE_SERVER_ON:-true}
-MESG_SERVICE_SERVER="engine-service-server"
-
-
 networkExists=$(docker network list -f name=engine -f driver=overlay -q)
 if [ "$networkExists" == "" ]; then
   docker network create --driver overlay engine --label com.docker.stack.namespace=engine
 fi
 
-function deleteServiceServer {
-  rm -rf tmp-systemservices
-  if [ "$MESG_SERVICE_SERVER_ON" = true ]; then
-    if docker service list | grep -q engine-service-server; then
-      docker service rm $MESG_SERVICE_SERVER
-    fi
-  fi
-}
-
-deleteServiceServer
-mkdir -p tmp-systemservices
-for i in systemservices/* ; do
-  if [ -d "$i" ]; then
-    echo "Creating archive for $i..."
-    cd $i
-    file=$(basename "$i")
-    tar -czf ../../tmp-systemservices/$file.tar.gz --exclude-from=.dockerignore .
-    varname="$(tr '[:lower:]' '[:upper:]' <<< ${file:0:1})${file:1}"
-    LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.${varname}URL=http://$MESG_SERVICE_SERVER/$file.tar.gz'"
-    cd ../..
+# from scripts/build-core.sh
+# upload system services
+for s in systemservices/* ; do
+  if [ -d "$s" ]; then
+    pushd $s > /dev/null
+    name=$(basename "$s")
+    varname="${name}"
+    LDFLAGS+=" -X 'github.com/mesg-foundation/core/config.${varname}Compiled=$(cat compiled.json)'"
```

Comment:
yes because of optimization but not only, we should only have the compiled version of the service and if we have the compiled version there is no need of having the sources of the system service in the engine and in that case, the compiled version is a snapshot of the exact version we want from this system service that we can manually update with something like 

```mesg-cli service:compile https://github.com/mesg-foundation/service-marketplace > systemservices/marketplace.json```